src/components/Navbar/index.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { ReactComponent as CartIcon } from "../../icons/cart-icon.svg";
```

Comment:
Use `index.js` in icons folder to export all svgs,
```suggestion
import { ReactComponent as CartIcon } from "../../icons";
```